pkg/ui/src/views/queries/statementDetails.tsx

```diff
@@ -0,0 +1,209 @@
+import d3 from "d3";
+import _ from "lodash";
+import React from "react";
+import { Helmet } from "react-helmet";
+import { connect } from "react-redux";
+import { RouterState } from "react-router";
+import { createSelector } from "reselect";
+
+import { refreshQueries } from "src/redux/apiReducers";
+import { AdminUIState } from "src/redux/state";
+import { statementAttr } from "src/util/constants";
+import { FixLong } from "src/util/fixLong";
+import { Duration } from "src/util/format";
+import { SqlBox } from "src/views/shared/components/sql/box";
+import { SummaryBar, SummaryHeadlineStat } from "src/views/shared/components/summaryBar";
+import * as protos from "src/js/protos";
+
+import { countBarChart, latencyBarChart } from "./barCharts";
+
+type StatementStatistics = protos.cockroach.sql.CollectedStatementStatistics$Properties;
+
+interface StatementDetailsOwnProps {
+  statement: StatementStatistics;
+  refreshQueries: typeof refreshQueries;
+}
+
+type StatementDetailsProps = StatementDetailsOwnProps & RouterState;
+
+interface NumericStat {
+  mean?: number;
+  squared_diffs?: number;
+}
+
+function variance(stat: NumericStat, count: number) {
+  return stat.squared_diffs / (count - 1);
+}
+
+interface NumericStatRow {
+  name: string;
+  value: NumericStat;
+}
+
+interface NumericStatTableProps {
+  rows: NumericStatRow[];
+  count: number;
+  format: (v: number) => string;
+}
+
+class NumericStatTable extends React.Component<NumericStatTableProps> {
+  static defaultProps = {
+    format: (v: number) => `${v}`,
+  };
+
+  render() {
+    return (
+      <table className="numeric-stats-table">
+        <thead>
+          <tr className="numeric-stats-table__row--header">
+            <th className="numeric-stats-table__cell" />
+            <th className="numeric-stats-table__cell">Mean</th>
+            <th className="numeric-stats-table__cell">Variance</th>
```

Comment:
Maybe this has already been designed or bikeshed elsewhere, but was there a reason to choose variance over std deviation? As a human looking at things like this I'm used to seeing and interpreting std deviation. If this stays as variance then I have to square root it in my head to get a gut feeling about what the number means.

Reply:
I think we just copied it over from what was in `crdb_internal.node_statement_statistics`; there hasn't been much of a discussion about it. Could easily square root to get stddev if that is easier to interpret.

Reply:
That'd be preferrable. Variance is easier to math. Stddev is easier for humans.

Reply:
Done.